xAPI.md

```diff
@@ -507,6 +507,37 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of xAPI. While a learner experiences a learning activity, data about that experience is tracked to a Learning Record Store (LRS). Authentication with the LRS may use the learner's credentials, the activity provider's credentials, or a combination. A Data Analysis Tool, acting as a Service with permission, can request the data from the LRS and use it to display meaningful reports or visualizations. In the case of more structured learning, a user may be dependent on an Activity Provider to deliver and/or authorize use of content to them. Once this triangle of authorization and access is complete, a registration exists for that user in that content tracked by that LRS and experience tracking may proceed.
```

Comment:
I think the problem here is we're trying to illustrate what is a little complex with a single simple diagram. The diagram and the last sentence here imply that the learner always has an account on the LRS, but that's often not the case. Often, the Activity Provider has credentials and makes statements about a number of users. Maybe we need two diagrams, one showing activity provider credentials being used and the other showing learner credentials. Aside from the last sentence I'm happy with the text in this section.